tests/layer_tests/tensorflow_tests/test_tf_Reverse.py

```diff
@@ -30,3 +31,49 @@ def create_reverse_net(self, shape, dims):
     def test_reverse_basic(self, params, ie_device, precision, ir_version, temp_dir):
         self._test(*self.create_reverse_net(**params),
                    ie_device, precision, ir_version, temp_dir=temp_dir)
+
+class TestReverseComplex(CommonTFLayerTest):
```

Comment:
please run this layer test locally on your machine. It should not be passing now because we have a build issue in the code above:)